probes/src/main/java/cloud/fogbow/probes/core/probes/fogbow/FogbowResourceAvailabilityProbe.java

```diff
@@ -48,35 +53,22 @@ protected void populateMetadata(Map<String, String> metadata, Pair<String, Float
         metadata.put(RESOURCE_LABEL, p.getKey().toLowerCase());
     }
 
-    private Pair<String, Float> getResourceAvailabilityValue(ResourceType type) {
+    private Pair<String, Float> getResourceAvailabilityValue(ResourceType type) throws Exception {
         LOGGER.debug("Getting audits from resource of type [" + type.getValue() + "]");
         Integer valueFailedAfterSuccessful = providerService
             .getAuditsFromResourceByState(OrderState.FAILED_AFTER_SUCCESSFUL_REQUEST, type,
                 lastTimestampAwake);
         Integer valueFulfilled = providerService
             .getAuditsFromResourceByState(OrderState.FULFILLED, type, lastTimestampAwake);
-        Float availabilityData = calculateAvailabilityData(valueFailedAfterSuccessful,
+        if(valueFailedAfterSuccessful == 0 && valueFulfilled == 0){
+            throw new Exception("Not found resource data to calculate resource availability.");
```

Comment:
It doesn't smell good to throw `Exception`. You can throw a specific exception.

Reply:
Created a new exception called OrdersStateChangeNotFoundException.